libraries/botbuilder-applicationinsights/botbuilder/applicationinsights/django/bot_telemetry_middleware.py

```diff
@@ -1,21 +1,21 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-import sys
-import json
+"""Bot Telemetry Middleware."""
+
 from threading import current_thread
 
 # Map of thread id => POST body text
-_request_bodies = {}
+_REQUEST_BODIES = {}
 
 def retrieve_bot_body():
     """ retrieve_bot_body
     Retrieve the POST body text from temporary cache.
-    The POST body corresponds with the thread id and should resides in 
+    The POST body corresponds with the thread id and should resides in
     cache just for lifetime of request.
 
     TODO: Add cleanup job to kill orphans

```

Comment:
Could we perhaps clarify this TODO? Is this completed or should we file a separate tracking bug for the future?

Reply:
Same as above..